src/cdk/collections/selection.ts

```diff
@@ -162,6 +164,13 @@ export class SelectionModel<T> {
       this._selection.forEach(value => this._unmarkSelected(value));
     }
   }
+
+  /** Throws an error if multiple values are passed into a selection model with a single value. */
+  private _warnMultipleValuesForSingleSelection(values: T[]) {
```

Comment:
I don't think the name is too accurate since it'll throw an error, not warn.

Reply:
Done.